client/components/bdConsistency.tsx

```diff
@@ -0,0 +1,74 @@
+"use client"
+
+import { getBDConsistency, getBDConsistency } from "@/actions/project"
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select"
+import { useEffect, useState } from "react"
+import BarGraph from "./barGraph"
+
+
+function BDConsistency(props: any) {
+  const {slug} = props;
+
+
+  type sprint = {
+    id: string,
+    value: string
+  }
+  const [sprints, setsprints] = useState<sprint[]>([{ id: '1', value: "sprint-1" }, { id: '2', value: "sprint-2" }, { id: '3', value: "sprint-3" }])
+  const [selectedSprintID, setselectedSprintID] = useState("")
+  const [showChart, setShowChart] = useState(true)
+  const [labels, setLabels] = useState<string[]>([])
+  const [open_points, setopen_points] = useState<number[]>([])
+  const [series, setSeries] = useState<any[]>([])
+
+  useEffect(()=>{
+    
+    getBDConsistency(slug).then((data)=>{
+
+      let sprintsCompletedData = data.filter((data: any)=> data.totalPoints !== 0);
+
+      sprintsCompletedData = sprintsCompletedData.sort(function (a: any, b: any) {
+        return ('' + a.sprintName).localeCompare(b.sprintName);
+    })
+      
+      let dataPoints = sprintsCompletedData.map((data: any) =>  data.totalPoints);
+
+      let labels = sprintsCompletedData.map((data: any) =>  data.sprintName);
+
+      let series = [
+        {
+            name: 'BDConsistency',
+            data: dataPoints
+        }
+      ]
+      setSeries(series)
+      setLabels(labels);
+      setShowChart(true)
+
+
+    });
+    
+
+  }, [])
+
+  return (
+    <div className="flex border-2 border-slate-300 rounded-md divide-x-2">
+      <div className="filters flex flex-col divide-y-2">
+        <div className="p-8 font-bold">Filters</div>
+        <div className="p-8">
+        </div>
+      </div>
+      {showChart ? <BarGraph name="BDConsistency"  labels={labels} series={series}/> : <div className="flex-1 p-16 min-h-50">Loading...</div>}
```

Comment:
It should be line graph instead of a bar graph as shown in the below image

<img width="1261" alt="image" src="https://github.com/AshankDsouza/SER516-Team-Kansas/assets/157257922/88634998-ae3d-4b03-84d8-151c5e595f42">